File: Data Structures & Algorithms/sort-colors/submission-0.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        vector<int> numArr(3);
        for(int i=0; i<nums.size(); i++){
            numArr[nums[i]]++;
        }
        int count = 0;
        for(int i=0; i<3; i++){
            for(int j=0; j<numArr[i]; j++){
                nums[count] = i;
                count++;
            }
        }
    }
};

File: Data Structures & Algorithms/count-connected-components/submission-0.py
class Solution:
    def countComponents(self, n: int, edges: List[List[int]]) -> int:
        
        adj = {i:[] for i in range(n)}
        for a,b in edges:
            adj[a].append(b)
            adj[b].append(a)
        
        visit = set()
        def dfs(i):
            visit.add(i)
            for nei in adj[i]:
                if nei not in visit:
                    dfs(nei)
        count = 0
        for i in range(n):
            if i not in visit:
                dfs(i)
                count+=1
        return count
            
            
